specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -256,18 +256,18 @@
                 }
             }
         },
```

Comment:
This is a good feedback, but patch won't be supported at first phase.
We are starting with the basic API, and there is full UX experience that is using it. We might extend it in the future if needed.

Regarding the PUT- it will require to send the entire resource each time, meaning that in your example at the third PUT it will unassign.